saleor/graphql/shop/mutations/refund_reason_reference_type_clear.py

```diff
@@ -0,0 +1,37 @@
+import graphene
+
+from ....permission.enums import SitePermissions
+from ...core import ResolveInfo
+from ...core.descriptions import ADDED_IN_322
+from ...core.doc_category import DOC_CATEGORY_ORDERS
+from ...core.mutations import BaseMutation
+from ...core.types.common import RefundReasonReferenceTypeClearError
+from ...site.dataloaders import get_site_promise
+from ..types import RefundSettings
+
+
+class RefundReasonReferenceTypeClear(BaseMutation):
+    refund_settings = graphene.Field(
+        RefundSettings, description="Refund settings.", required=True
+    )
+
+    class Meta:
+        description = (
+            "Updates RefundSettings. The `Page` (Model) Type will be cleared from `reasonReferenceType`. When it's cleared, passing reason reference to refund mutations is no longer required for staff user."
```

Comment:
Basically providing `reason reference` in such case will raise an error, right? So maybe it should be mentioned in description as well